Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift

```diff
@@ -820,6 +820,9 @@ package final class SwiftTargetBuildDescription {
         let scope = self.defaultBuildParameters.createScope(for: self.target)
         var flags: [String] = []
 
+        // A custom swift version.
+        flags += scope.evaluate(.SWIFT_VERSION)
```

Comment:
This should have `-swift-version` in front of it.

Reply:
Done!